authui/src/authflowv2/base.css

```diff
@@ -67,4 +67,10 @@
     --letter-spacing-label-large: 0.00625rem;
     --font-weight-label-large: 600;
   }
+
+  /* Alignment */
+  :root {
+    --alignment-branding: center;
```

Comment:
I suggest we keep the CSS property in the variable name, making it `branding-text-align: center`.

Reply:
@louischan-oursky 
The consideration here is it may not be only used in `text-align`, but also `align-items` `align-self` or other alignment related rules, depends on our implementation.

Reply:
1. Rename --alignment-branding to --alignment-logo.
2. Document --alignment-logo as a enum. Its valid values are `start`, `center`, `end`, and `hidden`.
3. Document --alignment-content as a enum. Its valid values are `start`, `center` and `end`.